index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="./styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">DR. John Maccarthur</h1>
```

Comment:
I think using two space indentation will make your code more legible and organised thus try to avoid the tab key. you can alternatively set the tab key to two spaces in your code editor preferences.

Reply:
- Your comment is very professional here, but In this case I would suggest advising the author to set up linters to easily catch such issues. It would enhance consistency in the format. What do you think?